docs/dev/README.md

```diff
@@ -0,0 +1,31 @@
+# Chef Infra Development Documentation
+
+This directory contains a collection of useful how-to guides for both new and seasoned Chef contributors. There are many guides explaining how to build, test, and contribute to Chef Infra as well as documents describing how the many subsystems function.
+
+A good first start is our [How Chef Infra Is Built](how_chef_is_built.md) and [Chef Infra Release and Support Schedule](release_and_support_schedule.md)
```

Comment:
how_chef_is_built.md leads to no where...?

Reply:
They both go to the same bogus location now